backend/web/workflows/linux_qc_patching_prep.py

```
@@ -0,0 +1,726 @@
+        # Provider method not yet implemented
+        return ExecuteQCStepResponse(
+            success=False,
+            instance_id=step_request.instance_id,
+            step=step_request.step,
+            status="not_implemented",
+            output=None,
+            error=str(e),
+            exit_code=None,
+            message=f"QC step execution not yet implemented for this provider",
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)